tests/unit/src/test/scala/tests/RenameLspSuite.scala

```diff
@@ -502,6 +502,17 @@ class RenameLspSuite extends BaseLspSuite("rename") {
     newName = "Animal"
   )
 
+  renamed(
+    "nested-symbol",
+    """|/a/src/main/scala/a/Main.scala
+       |package a
+       |object Foo {
+       |  object <<Ma@@in>>
+       |}
+       |""".stripMargin,
+    newName = "Child"
```

Comment:
Should I explicitly say here that the map of renamed files is empty, or can we assume that because of the parameter's default value?

Reply:
I would be explicit, since that’s the purpose of this test case

Reply:
I see there is one test already that passes `fileRenames = Map.empty`, so I'll do the same for explicitness.